packaging/bundle-validation/validate.sh

```diff
@@ -135,3 +167,15 @@ if [ "$?" -ne 0 ]; then
     exit 1
 fi
 echo "::warning::validate.sh done testing utilities slim bundle"
+
+if [[  $SPARK_HOME == *"spark-3.1"* ]]
```

Comment:
gonna remove this check after all base images uploaded. we should test any artifacts produced by any profile